conveyor/generator/state_machines/get_object/get_object.go

```diff
@@ -0,0 +1,199 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package getobject
+
+import (
+	"context"
+
+	"github.com/insolar/insolar/conveyor/fsm"
+	"github.com/insolar/insolar/conveyor/generator/generator"
+)
+
+type CustomEvent struct{}
+type CustomPayload struct{}
+type CustomAdapterResponsePayload struct{}
+type CustomAdapterHelper interface{}
+
+const (
+	InitState fsm.ElementState = iota
```

Comment:
ElementState is full state with fsm.ID
Use fsm.StateID here and check type in codegeneratoion